bugalink-frontend/pages/users/[id]/edit.tsx

```diff
@@ -85,9 +92,26 @@ export default function EditProfile({ data }) {
       setErrors(errors);
       if (Object.keys(errors).length === 0) {
         // Aquí puedes hacer la llamada a la API o enviar los datos a donde los necesites
-        console.log(
-          'Los datos del formulario son válidos. ¡Enviando formulario!'
-        );
+        const url = `users/${user.id}/edit`;
+
+        const formData = new FormData();
+        formData.append('photo', file);
+        formData.append('first_name', name);
+        formData.append('last_name', surname);
+
+        
+        axiosAuth
+          .put(url, formData)
+          .then((res) => {
```

Comment:
using async/await is often cleaner and prefered over `.then()` and `.catch()` as that's our code standard